src/renderer/components/Editor/TakeGroupComponent.tsx

```diff
@@ -118,12 +153,36 @@ const TakeGroupComponent = ({
   });
 
   return (
-    <CustomColumnStack sx={{ marginTop: '10px', marginBottom: '10px' }}>
-      <CustomRowStack sx={{ justifyContent: 'flex-end' }}>
-        {!isFirstTimeOpen && isTakeGroupOpened && <BlockIcon />}
-      </CustomRowStack>
-      {takes}
-    </CustomColumnStack>
+    <>
```

Comment:
Don't need this - fragments only needed when rendering multiple children at the top level